packages/javascript/bh-shared-ui/src/views/PrivilegeZones/History/HistoryContent.tsx

```diff
@@ -0,0 +1,285 @@
+// Copyright 2025 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { Card, CardHeader, CardTitle, DataTable, Skeleton } from '@bloodhoundenterprise/doodleui';
+import { type AssetGroupTagHistoryRecord } from 'js-client-library';
+import { DateTime } from 'luxon';
+import React, { useRef, useState } from 'react';
+import { useInfiniteQuery } from 'react-query';
+import { AppIcon } from '../../../components';
+import { SearchInput } from '../../../components/SearchInput';
+import { useTagsQuery } from '../../../hooks';
+import { DEFAULT_FILTER_VALUE, FilterDialog, type AssetGroupTagHistoryFilters } from './FilterDialog';
+import HistoryNotes from './HistoryNotes';
+import { useHistoryTableContext } from './HistoryTableContext';
+import { apiClient } from '../../../utils';
+
+const BASE_COLUMNS = [
+    {
+        header: () => <div className='pl-8 text-left'>Name</div>,
+        id: 'name',
+    },
+    {
+        header: () => <div className='pl-8 text-left'>Action</div>,
+        id: 'action',
+    },
+    {
+        header: () => <div className='pl-8 text-left'>Date</div>,
+        id: 'date',
+    },
+    {
+        header: () => <div className='pl-8 text-left'>Zone/Label</div>,
+        id: 'tier',
```

Comment:
Is this id still `tier`?

It would be so much easier to display the id name as the rendered text here and put it into a `.map` function.